### PR TITLE
Updating 0.016 to 0.1.0 for pinot-hadoop and docs

### PR DESCRIPTION
**Pull request by jenniferdai**

### CODE REVIEW

File: docs/pinot_hadoop.rst [hunk 1]
```diff
@@ -79,13 +79,13 @@ workflow to generate Pinot segments.
...
   mvn clean install -DskipTests -Pbuild-shaded-jar
-  hadoop jar pinot-hadoop-0.016-shaded.jar SegmentCreation job.properties
+  hadoop jar pinot-hadoop-0.1.0-shaded.jar SegmentCreation job.properties
 
 You can then use the SegmentTarPush job to push segments via the controller REST API.
 
 .. code-block:: none
 
-  hadoop jar pinot-hadoop-0.016-shaded.jar SegmentTarPush job.properties
+  hadoop jar pinot-hadoop-0.1.0-SNAPSHOT-shaded.jar SegmentTarPush job.properties
```
Comment (jackjlli) [inline docs/pinot_hadoop.rst]: Should the shaded jar names be the same, either `pinot-hadoop-0.1.0-shaded.jar` or `pinot-hadoop-0.1.0-SNAPSHOT-shaded.jar`?

Comment (snleee) [reply docs/pinot_hadoop.rst]: instead of `0.1.0`, we can put `<version>` since this will going to be changed periodically

Comment (jenniferdai) [reply docs/pinot_hadoop.rst]: when i generated, it became snapshot

Comment (jenniferdai) [reply docs/pinot_hadoop.rst]: ill put version here but leave it for the other one since the other docs are based on this version

**Pull request merged**